### PR TITLE
DisplayProperties: Auto-resolution and graphical fix

### PR DESCRIPTION
**Pull request by Quaker762**
A few fixes as recommended by @danboid in #556

### CODE REVIEW

File: Applications/DisplayProperties/DisplayProperties.cpp [hunk 1]
```diff
@@ -39,12 +39,37 @@ void DisplayPropertiesWidget::create_resolution_list()
...
     m_resolutions.append({ 1600, 900 });
     m_resolutions.append({ 1920, 1080 });
     m_resolutions.append({ 2560, 1080 });
 
+    Size find_size;
+
+    bool okay = false;
+    // Let's attempt to find the current resolution and select it!
+    find_size.set_width(m_wm_config->read_entry("Screen", "Width", "1024").to_int(okay));
+    if(!okay) {
```
Comment (awesomekling) [inline Applications/DisplayProperties/DisplayProperties.cpp]: clang-format
```suggestion
    if (!okay)
```

File: Applications/DisplayProperties/DisplayProperties.cpp [hunk 2]
```diff
@@ -39,12 +39,37 @@ void DisplayPropertiesWidget::create_resolution_list()
...
+        perror("DisplayProperties: failed to convert width to int!");
+        return;
+    }
+
+    find_size.set_height(m_wm_config->read_entry("Screen", "Height", "768").to_int(okay));
+    if(!okay) {
```
Comment (awesomekling) [inline Applications/DisplayProperties/DisplayProperties.cpp]:
```suggestion
    if (!okay) {
```

File: Applications/DisplayProperties/DisplayProperties.cpp [hunk 3]
```diff
@@ -39,12 +39,37 @@ void DisplayPropertiesWidget::create_resolution_list()
...
+        perror("DisplayProperties: failed to convert height to int!");
+        return;
+    }
+
+    int index = 0;
+    for(auto& resolution : m_resolutions) {
```
Comment (awesomekling) [inline Applications/DisplayProperties/DisplayProperties.cpp]:
```suggestion
    for (auto& resolution : m_resolutions) {
```

File: Applications/DisplayProperties/DisplayProperties.cpp [hunk 4]
```diff
@@ -39,12 +39,37 @@ void DisplayPropertiesWidget::create_resolution_list()
...
+        if(resolution == find_size) {
```
Comment (awesomekling) [inline Applications/DisplayProperties/DisplayProperties.cpp]:
```suggestion
        if (resolution == find_size) {
```

File: Applications/DisplayProperties/DisplayProperties.cpp [hunk 5]
```diff
@@ -39,12 +39,37 @@ void DisplayPropertiesWidget::create_resolution_list()
...
+    // Let's attempt to find the current resolution and select it!
+    find_size.set_width(m_wm_config->read_entry("Screen", "Width", "1024").to_int(okay));
+    if(!okay) {
+        perror("DisplayProperties: failed to convert width to int!");
+        return;
+    }
+
+    find_size.set_height(m_wm_config->read_entry("Screen", "Height", "768").to_int(okay));
+    if(!okay) {
+        perror("DisplayProperties: failed to convert height to int!");
```
Comment (awesomekling) [inline Applications/DisplayProperties/DisplayProperties.cpp]: `perror()` is a C library function that appends an explanation of the current `errno` value.
None of the preceding code sets `errno` meaningfully, so it's not appropriate to use `perror` here,

**Comment by Quaker762** [comment]
> Dispman? DisplayManager? DisplayProperties? Your 1AM is showing :D

yeah this was just before I passed out haha

> clang-format

I should alias `clang-format` to `clang-format -i`. I ran it, but missed the flag yikes.

File: Applications/DisplayProperties/DisplayProperties.cpp (hunk 5 above)
Comment (Quaker762) [reply Applications/DisplayProperties/DisplayProperties.cpp]: 1am mixup sorry. I'll change it to `fprintf(stderr, ...)`

**Pull request merged**